portfolio/src/main/java/com/google/sps/utils/ServletUtils.java

```diff
@@ -0,0 +1,23 @@
+package com.google.sps.utils;
+
+import javax.servlet.http.HttpServletRequest;
+
+/** Auxiliary functions and variables to support comment feature. */
+public class ServletUtils {
+    // Makes it non-initializable
+    private ServletUtils() {
+
```

Comment:
Nit: you can put {} on the same line

Reply:
Fixed; just separating them looks nicer for me haha